php_phongo.c

```diff
@@ -74,6 +74,8 @@
 
 #define PHONGO_DEBUG_INI "mongodb.debug"
 #define PHONGO_DEBUG_INI_DEFAULT ""
+#define PHONGO_METADATA_SEPARATOR " / "
+#define PHONGO_METADATA_SEPARATOR_LEN strlen(PHONGO_METADATA_SEPARATOR)
```

Comment:
Since `PHONGO_METADATA_SEPARATOR` is a string literal, `(sizeof(PHONGO_METADATA_SEPARATOR) - 1)` would be more commonly used in C. I believe PHP itself does that for various length constants.

This is definitely a nit, as I expect a compiler might optimize call to `strlen` out anyway.